pkg/epp/flowcontrol/controller/doc.go

```diff
@@ -18,105 +18,19 @@ limitations under the License.
 //
 // # Overview
 //
-// The `FlowController` is the central processing engine of the flow control system. It is a sharded, high-throughput
```

Comment:
All this content has been broken up and colocated near the relevant types / functions.